torch/csrc/api/include/torch/nn/functional/loss.h

```diff
@@ -26,6 +26,32 @@ inline Tensor cosine_embedding_loss(
       input1, input2, target, options.margin(), options.reduction());
 }
 
+inline Tensor multilabel_soft_margin_loss(
+    const Tensor& input,
+    const Tensor& target,
+    const MultiLabelSoftMarginLossOptions& options) {
```

Comment:
I think we can give `options` a default value like `options = {}`, because in Python people can call `F.multilabel_soft_margin_loss(input, target)` without setting the keyword arguments. (It would be great if we could add a test for the `multilabel_soft_margin_loss(input, target)` (i.e. no options) use case in C++ API as well. :D)